AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -7,6 +7,8 @@
 !
 ! SECTION: Popups - Regular rules
 !
+cjr.org###interContainer
+cjr.org###interVeil
```

Comment:
Looks like the ad leftover. Add the rules to the AdGuad Base - general_extensions.txt near the similar rules `#$#` with the pull request comment.
```
! https://github.com/AdguardTeam/AdguardFilters/pull/128485
cjr.org#$#html { overflow: auto !important; }
cjr.org#$##interContainer { display: none !important; }
cjr.org#$##interVeil { display: none !important; }
```